packages/alignments/src/SNPCoverageTrack/Tooltip.tsx

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React, { useRef } from 'react'
+import PropTypes from 'prop-types'
+import Popper from '@material-ui/core/Popper'
+import Paper from '@material-ui/core/Paper'
+import { observer } from 'mobx-react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Feature } from '@gmod/jbrowse-core/util/simpleFeature'
+
+const useStyles = makeStyles({
+  popper: {
+    fontSize: '0.8em',
+    zIndex: 1500, // important to have a zIndex directly on the popper itself, material-ui Tooltip uses popper and has similar thing
```

Comment:
```suggestion
    zIndex: theme.zIndex.tooltip, // important to have a zIndex directly on the popper itself, material-ui Tooltip uses popper and has similar thing
```

The theme defaults to 1500 anyway, but probably good to use the theme where possible.